src/tests/test_events/test_events.cpp

```diff
@@ -44,10 +44,10 @@ class TestEvents : public QObject
         QVERIFY(owner == m_properties);
     }
 
-    static void onPropertyChanged(mlt_properties owner, TestEvents* self, char* name)
+    static void onPropertyChanged(mlt_properties owner, TestEvents* self, mlt_event_data* data)
     {
         QVERIFY(self != nullptr);
-        QCOMPARE(name, "foo");
+		QCOMPARE(Mlt::EventData(data).to_string(), "foo");
```

Comment:
Did the whitespace get messed up here (and below)?

Reply:
yes, fixed locally